probe/overlay/weave_test.go

```diff
@@ -42,15 +41,18 @@ func TestWeaveTaggerOverlayTopology(t *testing.T) {
 const (
 	mockWeavePeerName     = "winnebago"
 	mockWeavePeerNickName = "winny"
+	mockResponse = `{
+		"router": {
+			"peers": [{
+				"name": "winnebago",
+				"nickname": "winny"
+			}]
+		}
+	}`
```

Comment:
Maybe remove the constants mockWeavePeer[Nick]Name if you're not using them in the response?